CONTRIBUTING.md

```diff
@@ -86,6 +87,9 @@ Official high quality brand logos and brand colors can usually be found in the f
 1. Wikimedia (which should provide a source)
 1. GitHub repositories
 
+It may be the case that official sources simply do not exist, but an unofficial icon has gained widespread acceptance and popularity. These icons can also be included, after being reviewed and accepted by maintainers on a case-by-base basis. Javascript is the prime example of this situation.
```

Comment:
A couple suggestions, some for brevity and some for clarity:

```suggestion
It may be the case that no official source exist, but an unofficial icon has gained widespread acceptance and popularity. In such cases the unofficial icon can be included, but the details will be judged on a case-by-base basis. The JavaScript icon is an example of this
```

Reply:
"exist" would also be changed to "exists"